.gitmodules

```diff
@@ -59,7 +59,7 @@
 	url = https://github.com/adafruit/Adafruit_CircuitPython_DotStar.git
 [submodule "ports/atmel-samd/peripherals"]
 	path = ports/atmel-samd/peripherals
-	url = https://github.com/adafruit/samd-peripherals.git
+	url = https://github.com/Bruin-Spacecraft-Group/bruinspace-samd-peripherals.git
```

Comment:
Please make a separate PR to update the Adafruit repo. We'll get that merged in before this PR.

Reply:
The PR got approved; I've updated the submodule reference to point to the merge commit that just got made